cognite/client/_version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "2.26.0"
+__version__ = "2.27.0"
```

Comment:
```suggestion
__version__ = "2.26.1"
```
Think this more of a patch than minor fix?